public/js/components/stories/Frames.js

```diff
@@ -28,8 +34,8 @@ function renderContainer(Component, frames) {
       margin: "auto",
       paddingTop: "100px"
     }},
-    dom.div({ style: { border: "1px solid #ccc", padding: "20px" }},
-            createElement(Provider, { store },
-                          createElement(Component)))
+    createElement(Provider, { store },
+      dom.div({className: "accordion"},
+        dom.div({className: "_content"}, createElement(Component))))
```

Comment:
this was needed for styles to apply. It *should* be a code smell that an element needs additional surrounding context to style correctly. I'll file a bug to fix this soon.